frontend/src/app/testQueue/QueueItem.tsx

```diff
@@ -151,6 +156,19 @@ const AreYouSure: React.FC<AreYouSureProps> = ({
 );
 Modal.setAppElement("#root");
 
+/* 
```

Comment:
@jeskew-gov I remember you having some things to say about what browsers could and couldn't do about TZ conversion, so I am imagining you may have opinions on this section. I may be confused, however.